compositor_render/src/renderer/texture/utils.rs

```diff
@@ -10,3 +13,15 @@ pub fn texture_size_to_resolution(size: &wgpu::Extent3d) -> Resolution {
         height: size.height as usize,
     }
 }
+
+pub fn sources_to_textures<'a>(sources: &'a [(&NodeId, &NodeTexture)]) -> Vec<Option<&'a Texture>> {
```

Comment:
inline that code, it's only used in 2 places
this is sth we want to avoid and eventually refactor out